### PR TITLE
Update EF version

### PR DESCRIPTION
**Pull request by csillikd**

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	</PropertyGroup>
 	<ItemGroup Label="Runtime Dependencies">
 		<PackageVersion Include="Funcky" Version="[3.0.0, 4)" />
-		<PackageVersion Include="Microsoft.EntityFrameworkCore" Version="[3.1.25, 8)" />
+		<PackageVersion Include="Microsoft.EntityFrameworkCore" Version="[8.0.8, 9)" />
```
Comment (Mafii) [inline Directory.Packages.props]: If possible, keep 3.1.25 as the minimum - does that work?

Comment (csillikd) [reply Directory.Packages.props]: Fixed

File: global.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
     "sdk": {
-        "version": "7.0.100",
+        "version": "8.0.100",
```
Comment (Mafii) [inline global.json]: 8.0.401? or 8.0.302?

Comment (Mafii) [reply global.json]: 100 is really old

Comment (csillikd) [reply global.json]: Addressed

**Review by Mafii — DISMISSED** [review]

**Review by Mafii — APPROVED** [review]

**Pull request merged**